e2e/playwright/editor-tests.spec.ts

```diff
@@ -670,14 +670,8 @@ a1 = startSketchOn(offsetPlane(XY, offset = 10))
 
     await scene.connectionEstablished()
 
-    // Wait for highlighting to kick in, a good proxy that the LSP is ready.
-    await expect
-      .poll(() =>
-        page.evaluate(
-          () => document.querySelector('.cm-line > span[class*="ͼ"]') !== null
-        )
-      )
-      .toBe(true)
+    // Wait for the linter diagnostics to populate, a good proxy that the LSP is ready.
+    await expect(page.locator('.cm-lint-marker-error')).toBeVisible()
```

Comment:
The syntax highlighting extension is now loaded immediately actually, while the LSP still needs a second to load, which broke this test. TODO: make the LSP load immediately, there's no reason it can't.